libraries/CurieBLE/src/internal/BLEDeviceManager.cpp

```diff
@@ -578,7 +578,7 @@ BLEDevice BLEDeviceManager::peripheral()
 
 bool BLEDeviceManager::startScanning()
 {
-    _adv_duplicate_filter_enabled = false;
```

Comment:
I don't think you can change code here. If you changed code here, it will make developer confusing. I think change the code in BLEDevice.cpp is more reasonable.